src/shared/components/case-editor/case-edit-submit/case-edit-submit.component.ts

```diff
@@ -263,11 +376,19 @@ export class CaseEditSubmitComponent implements OnInit, OnDestroy {
       .filter(cf => cf.show_summary_content_option);
   }
 
-  getCaseId(): String {
+  private getCaseFields(): CaseField[] {
+    if (this.caseEdit.caseDetails) {
+      return FieldsUtils.getCaseFields(this.caseEdit.caseDetails);
+    }
+
+    return this.eventTrigger.case_fields;
+  }
+
+  public getCaseId(): String {
```

Comment:
should this be string but not the String wrapper?

Reply:
Changed return type to `string`.